go-controller/pkg/node/gateway_iptables.go

```diff
@@ -438,7 +438,13 @@ func getGatewayIPTRules(service *kapi.Service, svcHasLocalHostNetEndPnt bool) []
 				}
 			}
 		}
-		for _, externalIP := range service.Spec.ExternalIPs {
+		externalIPs := make([]string, 0)
+		externalIPs = append(externalIPs, service.Spec.ExternalIPs...)
+		for _, ingress := range service.Status.LoadBalancer.Ingress {
+			externalIPs = append(externalIPs, ingress.IP)
+		}
+
+		for _, externalIP := range externalIPs {
```

Comment:
I wonder if we need to check `len(ingress.IP)>0` Like we do here -> https://github.com/ovn-org/ovn-kubernetes/blob/f8e36bfeb18df447b515ecead03075bf215dd669/go-controller/pkg/node/gateway_shared_intf.go#L226

Reply:
I'll add that, it's gonna be more bulletproof I guess.

Reply:
Done